x-pack/plugins/endpoint/public/applications/endpoint/components/header_nav.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { MouseEvent } from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { EuiTabs, EuiTab } from '@elastic/eui';
+import { useHistory, useLocation } from 'react-router-dom';
+
+export interface NavTabs {
+  name: string;
+  id: string;
+  href: string;
+}
+
+export const navTabs: NavTabs[] = [
+  {
+    id: 'home',
+    name: i18n.translate('xpack.endpoint.headerNav.home', {
+      defaultMessage: 'Home',
+    }),
+    href: '/',
+  },
+  {
+    id: 'management',
+    name: i18n.translate('xpack.endpoint.headerNav.management', {
+      defaultMessage: 'Management',
+    }),
+    href: '/management',
+  },
+  {
+    id: 'alerts',
+    name: i18n.translate('xpack.endpoint.headerNav.alerts', {
+      defaultMessage: 'Alerts',
+    }),
+    href: '/alerts',
+  },
+  {
+    id: 'policies',
+    name: i18n.translate('xpack.endpoint.headerNav.policies', {
+      defaultMessage: 'Policies',
+    }),
+    href: '/policy',
+  },
+];
+
+const Tabs = styled(EuiTabs)`
+  top: 1px;
+  &:before {
+    height: 0px;
+  }
+`;
+
+export const HeaderNavigation: React.FunctionComponent = React.memo(() => {
+  const history = useHistory();
+  const location = useLocation();
+
+  function renderNavTabs(tabs: NavTabs[]) {
+    return tabs.map((tab, index) => {
+      return (
+        <EuiTab
+          data-testid={`${tab.id}EndpointTab`}
+          data-test-subj={`${tab.id}EndpointTab`}
+          key={index}
+          href={`/app/endpoint${tab.href}`}
```

Comment:
🔴  You can't hardcode the `/app/endpoint` or else you break when kibana is running with base-path. I think @kqualters-elastic has added the use of [kibana-react](https://github.com/elastic/kibana/blob/master/src/plugins/kibana_react/README.md) to our application, wich means we can use their hook to retrieve coreStart and then get the base path from the `http` service (see this: https://github.com/elastic/kibana/pull/58288#discussion_r384520989).

An alternative for now is to add a `basePath` prop to this component and then send it in from `index.tsx` since you should have that info. available there.